code/script.js

```diff
@@ -1,51 +1,199 @@
+button.addEventListener('click', (event) => {
+  event.preventDefault();
+  console.log(event);
+  const userGuess = document.getElementById("name-input").value;
+  console.log(userGuess);
+  guess = userGuess;
+  nextQuestion();
+  nameInput.value = "";
+}, {once: true});
+
+
+const whatSize = () => {
+  questionNumber++
+  botSay(`Let's find out if "${guess}" is true! How smol are you?`);
+  inputWrapper.innerHTML = `
+  <div class=button-wrapper>
+<button id="btnBig">I'm a UNIT</button>
+<button id="btnAverage">Come closer!</button>
+<button id="btnSmall">Tiny, yet brave</button></div>
+  `
+  document
+    .getElementById("btnBig")
+    .addEventListener('click', () => nextQuestion("The biggest"), {once: true});
+  document
+    .getElementById('btnAverage')
+    .addEventListener('click', () => nextQuestion("Let's get comfy"), {once: true});
+  document
+    .getElementById('btnSmall')
+    .addEventListener('click', () => nextQuestion("So smol") , {once: true});
+  }
+
+
+const ifMonster = (size) => {
+  questionNumber++
+  console.log("howdy");
+  botSay(`Alright alright! ${size}, huh? Tell me, if you met a monster, what would you do?`);
+
+  if (size === "The biggest") {
+    inputWrapper.innerHTML = `
+    <radio>
+      <h5>Monster time!</h5>
+      <label><input type="radio" name="choice" id="badger" option value="fight">I fight! I win!</label>
+      <label><input type="radio" name="choice" id="hippo" option value="run">Dinner time!</label>
+    `
+
+    document.getElementById("badger").addEventListener("click", () => nextQuestion("Bring it on!"), {once: true});
+
+  document.getElementById("hippo").addEventListener('click', () => nextQuestion("Fear me, I'm mighty!"), {once: true});
+  
+
+  } else if (size === "Let's get comfy") {
+    inputWrapper.innerHTML = `
+      <radio>
+      <h5>Monster time!</h5>
+      <label><input type="radio" name="choice" id="facehugger" option value="hug">I hug it 🤗</label>
+      <label><input type="radio" name="choice" id="grandma" option value="cookie">I give it cookies 🍪🍪</label>
+    `
+
+  document.getElementById("facehugger").addEventListener("click", () => nextQuestion("Just within jumping distance..."), {once: true});
+
+  document.getElementById("grandma").addEventListener('click', () => nextQuestion("Aren't you a sweetheart?"), {once: true});
+
+  } else {
+    inputWrapper.innerHTML = `
+    <radio>
+    <h5>Monster time!</h5>
+    <label><input type="radio" name="choice" id="mosquito" option value="stab">I stab!</label>
+    <label><input type="radio" name="choice" id="shrimp" option value="punch">I punch!</label>
+    `
+
+  document.getElementById("mosquito").addEventListener("click", () => nextQuestion("Sluuurp..."), {once: true});
+  document.getElementById("shrimp").addEventListener("click", () => nextQuestion("BANG!"), {once: true});
+}
+}
+
+const finish = (monsterAnswer) => {
+  questionNumber++
+  soundEffectFinal();
+  inputWrapper.innerHTML = `
+  <h3>🌟Amazing!🌟</h3>`
+
```

Comment:
"Amazing" taking that big of space in the Chatbot was a nice choice. And adding the sound effects increased the engagement of it.